pyramid_oereb/lib/records/plr.py

```diff
@@ -169,11 +190,17 @@ def calculate(self, real_estate):
                 tested_geometries.append(geometry)
                 inside = True
         self.geometries = tested_geometries
+
+        # Points
+        nrOfPoints = self._sum_points()
+        self._nrOfPoints = nrOfPoints if nrOfPoints else None
```

Comment:
This if duplicate the one at line 155, no ?
Perhaps it's better to remove the one at line 155 (= `return points_to_sum`, so it returns 0 instead of "None", and this test give you "None")